scripts/install.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+
+DOWNLOAD_URL="https://github.com/redhat-developer/ocdev/releases/download/v0.0.1"
+OCDEV_VERSION="latest"
```

Comment:
where is this variable used, and what it is for?

Reply:
it's not used anywhere, removed